bin/extract_galaxy_tools.py

```diff
@@ -520,9 +540,8 @@ def get_tool_count_per_server(tool_ids: Any) -> pd.Series:
         tool_id_list = [x.strip(" ") for x in tool_ids.split(",")]
         data = check_tools_on_servers(tool_id_list)
         result_df: pd.DataFrame = pd.DataFrame()
-        result_df["true_count"] = data.sum(axis=1).astype(str)
-        result_df["false_count"] = len(data.columns)
-        result_df["counts"] = result_df.apply(lambda x: "({}/{})".format(x["true_count"], x["false_count"]), axis=1)
+        result_df["counts"] = data.sum(axis=1).astype(str)
+        result_df.loc["Tool count", "counts"] = len(data.columns)
```

Comment:
`Tool count` might be a bit vague. And probably better after `Galaxy tool ids` list